includes/definitions/keeneticos.yaml

```diff
@@ -0,0 +1,11 @@
+os: keeneticos
+group: keenetic
+text: 'KeeneticOS'
+type: network
+over:
+    - { graph: device_bits, text: 'Device Traffic' }
+    - { graph: device_mempool, text: 'Memory Usage' }
+discovery:
+    -
+        sysDescr:
+            - 'Keenetic'
```

Comment:
You could probably put this instead to make it clear you match everything starting with "Keenetic": 
```
discovery:
    -
        sysDescr_regex:
            - '/^Keenetic/'
```

Reply:
Thanks, corrected - interestingly I thought I'd get clever and try the below as a stable alternative:

```
          entPhysicalMfgName:
               - 'Keenetic'
```

It all worked in my discovery testing etc, but then in the tests `lnms dev:check -vv -o keeneticos` it went bonkers with errors matching on airos-af-ltu and several other things.

Reply:
Better to stick with sysDescr_regex, that will avoid any other SNMP request during discovery. sysDescr is always requested.